### PR TITLE
apply kmp-crypto version catalog

### PR DESCRIPTION
**Pull request by JesusMcCloud**
If Dr. Frankenstein was McGyver's apprentice… uses the kmp-crypto version catalog to avoid version mismatches.

directly targeting main for an xcframework release

### CODE REVIEW

File: vclib-openid/build.gradle.kts [hunk 1]
```diff
@@ -31,29 +35,37 @@ kotlin {
 
         commonTest {
             dependencies {
-                implementation("at.asitplus.wallet:eupidcredential:2.1.0-SNAPSHOT"){
-                    exclude(group = "org.jetbrains.kotlinx", module = "kotlinx-serialization-json") //TODO remove me once everything's updated to conventions with new serialization
+                implementation("at.asitplus.wallet:eupidcredential:2.1.0-SNAPSHOT") {
```
Comment (nodh) [inline vclib-openid/build.gradle.kts:38]: Could change this to 2.1.1

File: vclib-openid/build.gradle.kts [hunk 2]
```diff
@@ -31,29 +35,37 @@ kotlin {
...
+                    exclude(
+                        group = "org.jetbrains.kotlinx",
+                        module = "kotlinx-serialization-json"
+                    ) //TODO remove me once everything's updated to conventions with new serialization
                 }
                 implementation("at.asitplus.wallet:mobiledrivinglicence:1.0.0-SNAPSHOT")
```
Comment (nodh) [inline vclib-openid/build.gradle.kts:44]: could be changed to 1.0.0

**Review by nodh — COMMENTED** [review]
Doesn't look pretty, but if it works ...

File: conventions-vclib/src/main/resources/vcLibVersions.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+kmpCrypto=3.2.1
```
[the remark this answers is not in the document]
Comment (JesusMcCloud) [reply conventions-vclib/src/main/resources/vcLibVersions.properties:1]: It is also loaded in settings.gradle.kts, where no plugins are present. also: I forgot to externalise all other version strings, which i should do for consistency reasons

File: vclib-openid/build.gradle.kts (hunk 1 above)
Comment (JesusMcCloud) [reply vclib-openid/build.gradle.kts:38]: good thinking!

**Review by nodh — APPROVED** [review]

**Pull request merged**